app/views/transactions/_transactions_table.html.slim

```diff
@@ -15,7 +15,7 @@
     = paginate transactions, remote: true
   .row.col-sm-12
     = render 'home/flow' if params[:q]
-- elsif params[:q].blank?
+- elsif params[:q].blank? && search == false
```

Comment:
@Gataniel I think it will be better to use variable like `hide_search` and then use it `if local_assigns[:hide_search]`. In this case you may to not set this variable in render there it's not used.